packages/discovery-provider/ddl/functions/handle_comment_mention.sql

```diff
@@ -1,57 +0,0 @@
-create or replace function handle_comment_mention() returns trigger as $$
```

Comment:
i think you'd still want the to replace the existing function with a no-op? did @stereosteve say differently?

Reply:
@isaacsolo should have let you know that this was what steve said :)